assets/js/pluginsidebar/sidebar.jsx

```diff
@@ -110,7 +109,7 @@ function Sidebar() {
    * A helper function for setting selected sections.
    * @param {Array} next - The array of selected sections to set.
    */
-  const setSelectedSections = (next) => setSelectedSectionsRaw(JSON.stringify(next));
+  const setSelectedSections = (next) => setSelectedSectionsRaw(next);
 
```

Comment:
are these changes going to cause a problem with meta already set on a post as as json encoded string?

Reply:
I had a similar thought, and I don't know. It's worth checking before we release.